Documentation/dt-bindings.md

```diff
@@ -247,3 +247,40 @@ new keycode to the actual key.
 		};
 	};
 ```
+
+### GPIO leds
+
+We also support the `gpio-leds` node. This node follows upstream binding except
+that most properties are not supported. Currently only `gpios` and
+`default-state` are supported in lk2nd.
+
+```
+gpio-leds {
+    compatible = "gpio-leds";
+
+    red {
+        gpios = <&tlmm 3 GPIO_ACTIVE_HIGH>;
+        default-state = "on";
+    };
+
+    green {
+        gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+        default-state = "keep";
+    };
+
+    blue {
+        gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+        default-state = "off";
+    };
+};
+```
+
+This driver is mainly for lk1st since no previous boot-loader is capable to
+light up an LED in this case. This makes it unclear what state currently the
+device is in.  It's recommended to document all the LEDs here if lk1st is
+available for your device.
```

Comment:
> "It's recommended to document"

I'd actually word this more softly, like "if you wish to make sure some leds are enabled or disabled by lk1st on boot, you can use this driver"